test/e2e/subscription_e2e_test.go

```diff
@@ -3091,10 +3090,12 @@ func fetchSubscription(crc versioned.Interface, namespace, name string, checker
 	var lastCSV string
 	var lastInstallPlanRef *corev1.ObjectReference
 
-	err = wait.Poll(pollInterval, pollDuration, func() (bool, error) {
+	err := wait.Poll(pollInterval, pollDuration, func() (bool, error) {
+		var err error
 		fetchedSubscription, err = crc.OperatorsV1alpha1().Subscriptions(namespace).Get(context.Background(), name, metav1.GetOptions{})
-		if err != nil || fetchedSubscription == nil {
```

Comment:
removing the check for subscription == nil might mean the following bits can panic as they access innards

Reply:
Possible, but unlikely; if `err == nil` (i.e. success), then `fetchedSubscription` would be set. I can restore the check, but the action might be different.

Reply:
ack, seems unlikely